ext/storage_sources/s3_store/S3StoreLogSystem.cxx

```diff
@@ -0,0 +1,73 @@
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/AWSLogging.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+#include "S3StoreLogSystem.h"
+
+#include <cstdarg>
+
+using namespace Aws::Utils;
+using namespace Aws::Utils::Logging;
+
+S3StoreLogSystem::S3StoreLogSystem(WT_EXTENSION_API *wtApi, uint32_t awsVerbose)
+{
+    static const std::map<int32_t, Aws::Utils::Logging::LogLevel> verbosityMapping = {
```

Comment:
Suggestion: should we have this as a member of the class?

Reply:
@sulabhM @tammybailey, what do you think? I would rather have a static member.

Reply:
There's no reason to have it as a class member at the moment as we aren't using it anywhere else in the class. I think it's okay as is.

Reply:
I agree, the implementation meets the needs for now. We can change it in the future if required.